packages/core/src/amazonq/lsp/workspaceInstaller.ts

```diff
@@ -10,11 +10,12 @@ import { LanguageServerResolver } from '../../shared/lsp/lspResolver'
 import { Range } from 'semver'
 import { getNodeExecutableName } from '../../shared/lsp/utils/platform'
 import { fs } from '../../shared/fs/fs'
-import { cleanLspDownloads } from '../../shared'
+import { cleanLspDownloads, getLogger } from '../../shared'
 
 const manifestUrl = 'https://aws-toolkit-language-servers.amazonaws.com/q-context/manifest.json'
 // this LSP client in Q extension is only going to work with these LSP server versions
 const supportedLspServerVersions = '0.1.35'
+const logger = getLogger('amazonqWorkspaceLSP')
```

Comment:
`amazonqWorkspaceIndexingLsp` is too long, but probably "indexing" is the most important thing to call out?

```suggestion
const logger = getLogger('amazonqIndexingLsp')
```

Reply:
Right now, indexing is the only action supported by this lsp, but I could imagine Q adding a feature that does other "work" within the workspace. Also, I think it might be best to keep it inline with the `name` field we use in the resolver here: https://github.com/aws/aws-toolkit-vscode/blob/2157f55a3c8bc428a572c44c3f576a19e1eaa49a/packages/core/src/amazonq/lsp/workspaceInstaller.ts#L21

Perhaps we should pull these out somewhere to make them more discoverable, but that might involve a deeper refactor out of the scope here.